app/controllers/applications_controller.rb

```diff
@@ -3,7 +3,7 @@ class ApplicationsController < ApplicationController
 
   include ActionView::Helpers::DateHelper
 
-  ENVIRONMENTS = %w[production staging integration].freeze
+  ENVIRONMENTS = %w[production-aws staging-aws integration].freeze
```

Comment:
Quick fly-by comment: can we normalise this as `integration-aws` too? Seems a shame there's still an inconsistency even after this PR.

Reply:
Oh wait - looks like that would cause an issue here, wouldn't it? https://github.com/alphagov/govuk-app-deployment/pull/453/files

Reply:
So we migrated integration completely over to aws earlier than production/staging so we never really used `integration-aws`. However we never dropped the `-aws` suffix for production/staging, so deployments for them have continued to be `production-aws` and `staging-aws`.

This PR is scoped to removing the logic differences between deployments for Carrenza or AWS. After this PR, I'll raise another PR to make the environment name consistent i.e. `production`, `staging` and `integration` (dropping the `-aws`). But can't raise that until, this PR is merged. As part of dropping the `-aws` suffix, I will also merge the [change to app-deployments](https://github.com/alphagov/govuk-app-deployment/pull/453/files) so new deployments also don't use the `-aws` suffix.

TLDR; will ultimately end up with `production`, `staging` and `integration` - but just need to this as preliminary work, so I don't break Release and introduce too much downtime.